test/runtests.jl

```diff
@@ -3,7 +3,7 @@ using Dates
 using LinearAlgebra
 using Random
```

Comment:
Do failures go away if you add this here?
```julia
using Statistics
```

Reply:
No, the number of failures stays the same.